PrebidMobile/Rendering/PrebidMobileRendering/PrebidMobileRenderingTests/PBMFunctionsObjCTest.m

```diff
@@ -17,7 +20,7 @@ @implementation PBMFunctionsObjCTest
 #pragma mark - JSON
 
 - (void)testDictionaryFromDataWithNilData {
-    
+    /*
```

Comment:
Why did we comment these tests? If they are not relevant anymore - let's remove them. If we are going to restore them - need to uncomment them and disable in the project settings.

Reply:
This tests are commented as its can be compiled right now due import & redefinition (see the comment at the top of file). They should be fixed after full migration to Swift